docs/comparison.md

```diff
@@ -0,0 +1,139 @@
+# Comparison of `DataFrames` with R and Pandas
+
+R and Pandas (with Python) are commonly used data analysis tools. Both allow defining dataframes and provide a range of convenient ways of using them. Here, we translate some of the essential functionality of both languages. This is a good place to start if you know how to work with dataframes in R or Pandas and quickly want to get going using Julia.
+
+If you are instead interested in a general comparison of the languages, you can find that in the [base Julia documentation](http://docs.julialang.org/en/release-0.3/manual/noteworthy-differences/).
+
+For the examples below, we are following the [Pandas documentation](http://pandas.pydata.org/pandas-docs/stable/comparison_with_r.html) as much as possible. Since we use random numbers throughout, we assume that the ``Distributions`` package is loaded in Julia.
+
+# R
+
+## Indexing
+
+In R, columns of a `data.frame` can be accessed by name:
+
+```R
+df <- data.frame(a=rnorm(5), b=rnorm(5), c=rnorm(5), d=rnorm(5), e=rnorm(5))
+df[, c("a", "c")]
+
+df <- data.frame(matrix(rnorm(1000), ncol=100))
+df[, c(1:10, 25:30, 40, 50:100)]
+```
+
+Using DataFrames, the equivalent would be the following:
+
+```julia
+d = Uniform()
+df = DataFrame(a=rand(d, 5), b=rand(d, 5), c=rand(d, 5), d=rand(d, 5), e=rand(d, 5))
+df[[:a, :c]]
```

Comment:
How about comparing apples to apples -- `rnorm` to `randn(5)`, `runiform` to `rand(10, 10)`, etc.,  throughout the code, and later on mix in a less common distribution, `Exponential` or anything else, with the `using Distributions` (instead of in the text above) and a comment that `Distributions` also includes `Normal()` and `Uniform()`?